integration-tests/kafka-streams/src/test/java/io/quarkus/it/kafka/streams/KafkaStreamsTest.java

```diff
@@ -124,7 +124,8 @@ public void testKafkaStreamsNotAliveAndNotReady() throws Exception {
                 .statusCode(HttpStatus.SC_SERVICE_UNAVAILABLE)
                 .body("checks[0].name", CoreMatchers.is("Kafka Streams topics health check"))
                 .body("checks[0].status", CoreMatchers.is("DOWN"))
-                .body("checks[0].data.missing_topics", CoreMatchers.is("streams-test-customers,streams-test-categories"));
+                .body("checks[0].data.missing_topics",
```

Comment:
Why is the processed topic expected now? It somehow seems that now also sink topics are awaited? If so, I don't think that'd be adviseable, as they only might be created by running the app itself, causing a dead-lock effectively.